docs/advanced-topics/wes/wes-agc-tutorial.rst

```diff
@@ -0,0 +1,329 @@
+WES tutorial for AWS AGC
+========================
+Amazon Web Services' (AWS) Amazon Genomics CLI (AGC) is a command line tool for launching cloud infrastructure
+within AWS accounts that can be used to execute genomics workflows. The infrastructure deployed by AGC implements the WES
+standard, and thus can be directly communicated with by using the Dockstore CLI.
+
+Check out the official AGC `GitHub page <https://github.com/aws/amazon-genomics-cli>`_.
+
+Download and install AWS AGC
+----------------------------
+AGC provides a `quick-start guide <https://aws.github.io/amazon-genomics-cli/docs/getting-started/>`_ for initial setup
+and getting familiar with the tool. The following workflow execution tutorials will cover all steps for using AGC once
+it has been `installed <https://aws.github.io/amazon-genomics-cli/docs/getting-started/installation/>`_.
+
+
+
+
+Tutorial topics
+----------------
+The following WDL workflow tutorials will covering deploying the proper infrastructure using AGC.
+
+1. Deploy an AGC project and context
+2. Configure the Dockstore CLI to communicate with AGC infrastructure
+3. Launching a workflow
+
+.. note::
+
+    1. AWS AGC requires a special JSON file to passed as the test parameter file for workflows using the format
+
+        .. code:: text
+
+            {
+                "workflowInputs": <pathToInputJSON>
+            }
+
+    2. AWS AGC references descriptor files by URLs. The Dockstore CLI communicates to AGC infrastructure using TRS URLs,
```

Comment:
May want to describe, or at least spell out TRS. And preceded with GA4GH.

Also "communicates to AGC infrastructure using TRS URLs" doesn't quite sound right to me. Makes WES requests with references to TRS URLs, maybe something like that.

Reply:
Reworded the majority of the bullet: https://github.com/dockstore/dockstore-documentation/pull/173/commits/168253455ed40e8abaee72f12ae16603d9b3e759 and then reworded again, slightly: https://github.com/dockstore/dockstore-documentation/pull/173/commits/070439122feb9758b2bfe4e0ddcb8fab4d725d5c